test/batch/configuration.ts.snap

```diff
@@ -0,0 +1,92 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Configuration Configuration is invalid if check_type is missing 1`] = `
```

Comment:
is it OK to have doubled `Configuration Configuration` in this file?

Reply:
The titles can easily be improved through code tweaks. I've opened #44 for that.